src/Playroom/Toolbar/Toolbar.tsx

```diff
@@ -157,13 +158,14 @@ export default ({ themes: allThemes, widths: allWidths, snippets }: Props) => {
         </div>
         <CSSTransition
           in={isOpen}
+          nodeRef={nodeRef}
```

Comment:
To do this you dont need to use `ReactDOM.findDOMNode` which is removed in react@19
https://github.com/reactjs/react-transition-group/blob/2989b5b87b4b4d1001f21c8efa503049ffb4fe8d/src/ReplaceTransition.js#L32-L35